AutomatedTesting/Gem/PythonTests/prefab/Prefab_Test_Utils.py

```diff
@@ -0,0 +1,94 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+
+from azlmbr.entity import EntityId
+from azlmbr.math import Vector3
+from editor_python_test_tools.editor_entity_utils import EditorEntity
+from editor_python_test_tools.utils import Report
+from editor_python_test_tools.utils import TestHelper as helper
+
+import azlmbr.bus as bus
+import azlmbr.components as components
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+
+def get_prefab_file_name(prefab_name):
+    return prefab_name + ".prefab"
+
+def get_prefab_file_path(prefab_name):
+    return os.path.join(os.path.dirname(os.path.abspath(__file__)), get_prefab_file_name(prefab_name))
+
+def find_entities_by_name(entity_name):
+    searchFilter = entity.SearchFilter()
+    searchFilter.names = [entity_name]
+    return entity.SearchBus(bus.Broadcast, 'SearchEntities', searchFilter)
+
+def find_entity_by_unique_name(entity_name):
+    unique_name_entity_found_result =  (
+        "Entity with a unique name found",
+        "Entity with a unique name *not* found")
+
+    entities = find_entities_by_name(entity_name)
+    unique_name_entity_found = len(entities) == 1
+    Report.result(unique_name_entity_found_result, unique_name_entity_found)
+
+    if unique_name_entity_found:
+        return entities[0]
+    else:
+        Report.info(f"{len(entities)} entities with name '{entity_name}' found")
+        return EntityId()
+
+def check_entity_at_position(entity_id, expected_prefab_position):
```

Comment:
nit: This function can be used to provide any position. So maybe expected_entity_position is a better name here.

Reply:
Fixed.